inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/SubclassCreationDialog.java

```diff
@@ -128,10 +129,20 @@ protected void actionCreateSubclass(AjaxRequestTarget aTarget, Form<KBConcept> a
 
                 // get subclassof property
                 KnowledgeBase kb = kbModel.getObject();
-                KBProperty property = kbService.readProperty(kb, kb.getSubclassIri().stringValue())
-                        .get();
-                KBHandle propertyHandle = new KBHandle(property.getIdentifier(), property.getName(),
-                        property.getDescription());
+                Optional<KBProperty> propertyContainer = kbService.readProperty(kb,
```

Comment:
Again there is no explanatory comment here.

Reply:
Also, the PR doesn't include a test case.

Reply:
I suppose we didn't have these problems when the inferencer was still enabled because it would have initialized the KB with the basic RDF and OWL IRIs. Maybe it would be a good idea that we prep freshly created local KBs with a few basic things to start with, e.g. the foundational RDF, RDFS and OWL classes and properties?

Reply:
Yeah , we have these problems after we removed the inferencer. As we dont have the base properties such as `subClassOf`. 

>  Maybe it would be a good idea that we prep freshly created local KBs with a few basic things to start with, e.g. the foundational RDF, RDFS and OWL classes and properties?

In that case, we can just initialize the basic properties with out actually tagging them to class instances. The list can be : 
- subClassOf
- instanceOf
WDYS, do we need any other property?

Reply:
But the idea is where should we actually do that, should we do that when we read the concept tree for the first time? OR in listProperty ?

Reply:
We will actually then try to replicate the basic inferencer. Need to check if that would create any issues with the medium sized KB's